lib/include/jxl/decode.h

```diff
@@ -223,19 +223,19 @@ typedef enum {
    * JxlDecoderGetFrameHeader can be used at this point. A note on frames:
    * a JPEG XL image can have internal frames that are not intended to be
    * displayed (e.g. used for compositing a final frame), but this only returns
-   * displayed frames, unless @ref JxlDecoderSetCoalescing was set to JXL_FALSE:
-   * in that case, the individual layers are returned, without blending. Note
-   * that even when coalescing is disabled, only frames of type kRegularFrame
-   * are returned; frames of type kReferenceOnly and kLfFrame are always for
-   * internal purposes only and cannot be accessed. A displayed frame either has
-   * an animation duration or is the only or last frame in the image. This event
-   * occurs max once per displayed frame, always later than @ref
-   * JXL_DEC_COLOR_ENCODING, and always earlier than any pixel data. While
-   * JPEG XL supports encoding a single frame as the composition of multiple
-   * internal sub-frames also called frames, this event is not indicated for the
-   * internal frames.
-   * In this case, @ref JxlDecoderReleaseInput will return all bytes from the
-   * end of the frame header (including ToC) as unprocessed.
+   * displayed frames, unless @ref JxlDecoderSetCoalescing was set to @ref
+   * JXL_FALSE "JXL_FALSE": in that case, the individual layers are returned,
+   * without blending. Note that even when coalescing is disabled, only frames
+   * of type kRegularFrame are returned; frames of type kReferenceOnly and
```

Comment:
Somehow ref kConstants as well?

Reply:
I could add @ref to all of the constants, where those defined are in some of the files where we build doxygen files. So it works for all, except `kRegularFrame` and `kReferenceOnly`, because those are defined in frame_header.h which is apparently not build in doxygen. 
(Should we move the definition of the enum FrameType to a more public place?)